### DISCUSSION TITLE
Optimize performance for Chrome (Firefox partially 2 times faster)

### DISCUSSION THREAD

**Discussion by svenpilz** [emscripten-core/emscripten#15915] [Q&A]
Hi fellow Emscripten users,

how do your builds perform on Chrome? We noticed that the exact same binaries are up to 2 times faster when running in Firefox. Is there a way to tweak Chrome via JS to optimize more during WASM-to-x86 compilation?

Greetings from Berlin
Sven

**Comment by kripken** [comment]
There are sometimes differences between browsers but a 2x difference is very surprising. If you can provide a testcase, we can file a bug for Chrome, hopefully this is fixable.

There aren't really any settings or tricks to optimize for a particular browser, except to perhaps work around a particular browser performance bug. But such bugs come and go and are hard to predict.
